gtsam/CMakeLists.txt

```diff
@@ -199,7 +199,7 @@ if(WIN32)
 else()
   if("${CMAKE_BUILD_TYPE}" STREQUAL "Release")
     # Suppress all warnings from 3rd party sources.
-    set_source_files_properties(${3rdparty_srcs} PROPERTIES COMPILE_FLAGS "-w")
+    set_source_files_properties(${3rdparty_srcs} PROPERTIES COMPILE_FLAGS "-w -Wno-everything")
```

Comment:
what's this?

Reply:
That's the flag to suppress warnings in Clang. `-w` already does so for GCC, but Clang still throws warnings in Release mode.